nxdrive/engine/engine.py

```diff
@@ -448,12 +448,16 @@ def _save_last_dt_session_infos(
             )
 
     def _create_remote_folder(
-        self, remote_parent_path: str, new_folder: str, /
+        self, remote_parent_path: str, new_folder: str, session_id: int, /
     ) -> Dict[str, Any]:
         try:
             return self.remote.upload_folder(
                 remote_parent_path,
                 {"title": new_folder},
+                headers={
+                    "NX-metric-directTransfer.option.newFolder": str(True),
```

Comment:
Should we use a simple integer intead? It would be simpler to filter on it then.